cmd/minishift/cmd/start.go

```diff
@@ -187,13 +189,21 @@ func runStart(cmd *cobra.Command, args []string) {
 		atexit.Exit(1)
 	}
 
+	automountHostfolders(host.Driver)
+
 	clusterUp(&config, ip)
 	if err := openshift.AddSudoersRoleForUser("developer"); err != nil {
 		glog.Errorln("Error adding developer user to sudoers", err)
 		atexit.Exit(1)
 	}
 }
 
+func automountHostfolders(driver drivers.Driver) {
+	if hostfolder.IsAutoMount() && hostfolder.IsHostfoldersDefined(false) {
+		hostfolder.MountHostfolders(driver)
```

Comment:
I actually created a branch: https://github.com/gbraad/minishift/commit/0f4f89f748f61c248ab06ed5f8f89253a1572ed1

and I still need the driver to determine the hostip. I can not get the sshcommander to return the driver needed for this. Although it is public: https://github.com/docker/machine/blob/e1a03348ad83d8e8adb19d696bc7bcfb18ccd770/libmachine/provision/generic.go#L28 I do not seems to get access to it due to the interface: https://github.com/docker/machine/blob/39e78dff7df191501a034282366c82e4bcbfacdb/libmachine/provision/provisioner.go#L24